pytorch_lightning/accelerators/accelerator.py

```diff
@@ -383,13 +375,6 @@ def setup_precision_plugin(self) -> None:
         self.optimizers = optimizers
         self.schedulers = schedulers
 
-    def to_device(self, step_kwargs: Dict[str, Union[Any, int]]) -> Dict[str, Union[Any, int]]:
-        """Pushes the batch to the root device"""
-        step_kwargs['batch'] = self.batch_to_device(
-            step_kwargs['batch'], self.root_device, dataloader_idx=step_kwargs.get('dataloader_idx', None)
-        )
-        return step_kwargs
```

Comment:
just fyi:
there's a to_device method in `accelerators/gpu.py`, you might need to take care of that too and update accordingly.

Reply:
thanks, didn't see that.